lib/card/index.js

```diff
@@ -33,12 +34,10 @@ class Card extends React.Component {
       );
     }
 
+    const Icon = expanded ? FaAngleDoubleUp : FaAngleDoubleDown;
```

Comment:
- Use a single icon component and use transform to rotate it, it will ensure the expanding of card body and change of icon is together.

Reply:
- Remove little bit of space from bottom of Expand/Collapse icon.
- https://disjfa.github.io/bootstrap-tricks/card-collapse-tricks/ Refer this URL to see the smoothness in card expansion and icon rotation example